src/root-config-guide/root-config-guide.component.js

```diff
@@ -0,0 +1,192 @@
+          is actually the origin of the single-spa name (only a "single"
+          single-page-application -- don't think about it too hard{" "}
+          <span role="img" aria-label="Smiley emoji">
+            😀
+          </span>
+          )
+        </section>
+        <section>
+          The single-spa root config is something that we can provide for you
+          entirely. Let's use the create-single-spa again!
+        </section>
+        <h2>Creating a single-spa root config</h2>
+        <div>
+          You'll want to create an{" "}
+          <span className="whole-new-git-repo">entirely new git repo</span> for
+          your single-spa root config, separated from your applications.
+        </div>
+        <div>
+          <Code code={`npx create-single-spa --module-type root-config`} />
+        </div>
+        <p>
+          This command won't give you a running root-config out of the box for
+          our playground demonstration! But, fear not, you will understand
+          everything root-config related now!
+        </p>
+        <p>
+          First, lets talk about the{" "}
+          <a href="https://github.com/WICG/import-maps" target="_blank">
+            import maps
+          </a>
+          !
+        </p>
+        <div style={{ display: "flex", justifyContent: "center" }}>
+          <iframe
+            title="import-map-tutorial"
+            width="560"
+            height="315"
+            src="https://www.youtube.com/embed/Lfm2Ge_RUxs"
+            frameBorder="0"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen
+          ></iframe>
+        </div>
+        <p>
+          Import maps are a browser specification for controlling which{" "}
+          <b>URL to download javascript bundles from</b>. The npm library{" "}
+          <a href="https://github.com/systemjs/systemjs" target="_blank">
+            systemjs
+          </a>{" "}
+          is a polyfill for this specification, which lets us use import maps in
+          browsers that don't yet support them. We will use an import map to
+          control where to download your applications from, which sets you up
+          for easy independent deployability once you get things working
+          locally.
+        </p>
+        <p>
+          If you've been following this guide, and created a single-spa react
+          application using our CLI (create-single-spa), how about checking the{" "}
+          <a
+            href={
+              window.importMapOverrides.getOverrideMap().imports[
+                application.name
+              ]
+            }
+            target="_blank"
+          >
+            JS generated by webpack now?
+          </a>{" "}
+          What is important for now, is this line:
+        </p>
+        <Code code={`System.register(["react","react-dom"], function(...`} />
```

Comment:
Earlier in the tutorial, it says this would have a "react focus." But also we have tried to extract the framework by running a regex on the code - could we make this more specific to the framework they're using, if we know it?